curl_cffi/__init__.py

```diff
@@ -1,21 +1,28 @@
 __all__ = [
     "Curl",
+    "CurlChrome",
+    "CurlFirefox",
     "CurlInfo",
     "CurlOpt",
     "CurlMOpt",
     "CurlECode",
     "CurlHttpVersion",
     "CurlError",
     "AsyncCurl",
+    "AsyncCurlChrome",
+    "AsyncCurlFirefox",
     "ffi",
     "lib",
+    "ffi_ff",
+    "lib_ff",
 ]
 
-# This line includes _wrapper.so into the wheel
-from ._wrapper import ffi, lib
+# This line includes _wrapper_{chrome,ff}.so into the wheel
+from ._wrapper_chrome import ffi, lib
+from ._wrapper_ff import ffi as ffi_ff, lib as lib_ff
 
 from .const import CurlInfo, CurlMOpt, CurlOpt, CurlECode, CurlHttpVersion
-from .curl import Curl, CurlError
-from .aio import AsyncCurl
+from .curl import Curl, CurlChrome, CurlFirefox, CurlError
+from .aio import AsyncCurl, AsyncCurlChrome, AsyncCurlFirefox
```

Comment:
Would it need to deprecate `Curl` and `AsyncCurl` in favor of `CurlChrome` and `AsyncCurlChrome`?